Mathlib/CategoryTheory/Abelian/SerreClass/Basic.lean

```diff
@@ -64,6 +65,10 @@ lemma prop_X₂_of_exact {S : ShortComplex C} (hS : S.Exact)
   exact (P.prop_X₂_of_shortExact (hS.shortExact d)
     (P.prop_of_epi d.left.f' h₁) (P.prop_of_mono d.right.g' h₃) :)
 
+instance (F : D ⥤ C) [F.Additive] [PreservesFiniteLimits F]
+    [PreservesFiniteColimits F] :
+    (P.inverseImage F).IsSerreClass where
```

Comment:
In additive categories, either one of `PreservesFiniteLimits` and `PreservesFiniteColimits` implies `Additive`, so you should be able to drop that assumption.